Update validator FAQ (#495)
lukso-network/docs: docs/faq/network/network.md

---
title: General
description: LUKSO General Network FAQ
sidebar_position: 1
---

# Frequently Asked Questions

## General

### What are the expected block and epoch times?

The LUKSO network will run with an expected block time of 12 seconds with 12 seconds per slot. An epoch contains 32 slots.

### Which cold storage wallets can be used with LUKSO network?

> LUKSO is an EVM compatible network. Therefore, any Ethereum compatible wallet which allows adding custom networks can be used on the LUKSO network.

Reviewer: ```suggestion
LUKSO is an EVM-compatible network. Therefore, any Ethereum-compatible wallet allowing custom networks can be used on the LUKSO network.
```

### Will LUKSO implement the EIP-1559 standard?

Yes, LUKSO will launch with EIP-1559 enabled.

### What will the LUKSO network token distribution voting process look like in practice?

> When using our genesis validator deposit launchpad, validators will be able to select their preferred token distribution from **35M, 42M or 100M LYX** tokens. The network will start with the option most voted for by genesis validators.

Reviewer: ```suggestion
When using our genesis validator deposit launchpad, validators can select their preferred token distribution from **35M, 42M, or 100M LYX** tokens. The network will start with the option most voted for by genesis validators.
```

### Will the test network be open to community validators?

The LUKSO test network will be open to whitelisted validator addresses only.

### Will there be a guaranteed minimum amount of time that the discovery phase will last before non-genesis validators can be added?

> The length of the discovery phase will depend on several factors including the number of genesis validators and the overall stability of the network. We will move on to the next phase once we are satisfied everything is running smoothly. It is expected to last around 1 month.

Reviewer: ```suggestion
The length of the discovery phase will depend on several factors, including the number of genesis validators and the overall stability of the network. Once we are satisfied everything is running smoothly, we will move on to the next phase. It is expected to last around 1 month.
```

### How can I transfer my LYXe to LYX

> Details about the LYXe to LYX migration process for those who do not participate as genesis validators will come later, after the genesis and discovery phase of the network have completed.

Reviewer: ```suggestion
Details about the LYXe to LYX migration process for those who do not participate as genesis validators will come later after the genesis and discovery phase of the network have been completed.
```
